app/src/androidTest/java/com/github/brugapp/brug/ItemsMenuActivityTest.kt

```diff
@@ -56,8 +59,9 @@ class ItemsMenuActivityTest {
     /* THESE NEED A CONFLICTING ESPRESSO DEPENDENCY */
     @Test
     fun swipeLeftOnItemTriggersSnackBar() {
+        Intents.init()
```

Comment:
Do you plan on adding new tests that use Intents in the future ?
If yes, it would be great to put the Intents.init() and Intents.release() functions into a @Before  and @After function respectively, to avoid code duplication and conflicts. You can keep it in mind for next time or change it now as it is not a problem yet.

```suggestion
// (outside the class)
@Before
fun setUp() {
    Intents.init()
}
```